code/_onclick/hud/hud.dm

```diff
@@ -15,9 +15,22 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 	"Glass" = 'icons/hud/screen_glass.dmi'
 ))
 
+GLOBAL_LIST_INIT(available_erp_ui_styles, list(
```

Comment:
Remember to add //SKYRAT EDITS on the bits you've changed

Reply:
here is only additions.
edits only in previous blocks that have that comments